src/functions-reference/distributions_over_unbounded_vectors.Rmd

```diff
@@ -593,6 +594,139 @@ freedom nu, locations mu, and scale matrix Sigma; may only be used in
+`r since("2.30")`
+
+<!-- real; multi_student_t_cholesky_lpdf; (row_vectors y | real nu, row_vectors mu, matrix L); -->
+\index{{\tt \bfseries multi\_student\_t\_cholesky\_lpdf }!{\tt (row\_vectors y \textbar\ real nu, row\_vectors mu, matrix L): real}|hyperpage}
+
+`real` **`multi_student_t_cholesky_lpdf`**`(row_vectors y | real nu, row_vectors mu, matrix L)`<br>\newline
+The log of the multivariate Student-$t$ density of row vector(s) y
+given degrees of freedom nu, location row vector(s) mu, and Cholesky factor of the scale matrix L
+`r since("2.30")`
+
+<!-- real; multi_student_t_cholesky_lupdf; (row_vectors y | real nu, row_vectors mu, matrix L); -->
+\index{{\tt \bfseries multi\_student\_t\_cholesky\_lupdf }!{\tt (row\_vectors y \textbar\ real nu, row\_vectors mu, matrix L): real}|hyperpage}
+
+`real` **`multi_student_t_cholesky_lupdf`**`(row_vectors y | real nu, row_vectors mu, matrix L)`<br>\newline
+The log of the multivariate Student-$t$ density of row vector(s) y
+given degrees of freedom nu, location row vector(s) mu, Cholesky factor of the scale matrix L
+dropping constant additive terms
+`r since("2.30")`
+
+<!-- vector; multi_student_cholesky_t_rng; (real nu, vector mu, matrix L); -->
+\index{{\tt \bfseries multi\_student\_t\_cholesky\_rng }!{\tt (real nu, vector mu, matrix L): vector}|hyperpage}
+
+`vector` **`multi_student_t_cholesky_rng`**`(real nu, vector mu, matrix L)`<br>\newline
+Generate a multivariate Student-$t$ variate with degrees of freedom
+nu, location mu, and Cholesky factor of the scale matrix L; may only be used in transformed data
+and generated quantities blocks
+`r since("2.30")`
+
+<!-- vector; multi_student_t_cholesky_rng; (real nu, row_vector mu, matrix L); -->
+\index{{\tt \bfseries multi\_student\_t\_cholesky\_rng }!{\tt (real nu, row\_vector mu, matrix L): vector}|hyperpage}
+
+`vector` **`multi_student_t_cholesky_rng`**`(real nu, row_vector mu, matrix L)`<br>\newline
+Generate a multivariate Student-$t$ variate with degrees of freedom
+nu, location mu, and Cholesky factor of the scale matrix L; may only be used in transfomed data
+and generated quantities blocks
+`r since("2.30")`
+
+<!-- vectors; multi_student_t_cholesky_rng; (real nu, vectors mu, matrix L); -->
+\index{{\tt \bfseries multi\_student\_t\_cholesky\_rng }!{\tt (real nu, vectors mu, matrix L): vectors}|hyperpage}
+
+`vectors` **`multi_student_t_rng`**`(real nu, vectors mu, matrix L)`<br>\newline
+Generate an array of multivariate Student-$t$ variates with degrees of
+freedom nu, locations mu, and Cholesky factor of the scale matrix L; may only be used in
+transformed data and generated quantities blocks
+`r since("2.30")`
+
+<!-- vectors; multi_student_t_rng; (real nu, row_vectors mu, matrix Sigma); -->
+\index{{\tt \bfseries multi\_student\_t\_cholesky\_rng }!{\tt (real nu, row\_vectors mu, matrix L): vectors}|hyperpage}
+
+`vectors` **`multi_student_t_cholesky_rng`**`(real nu, row_vectors mu, matrix L)`<br>\newline
```

Comment:
I pushed an update for this. I feel like this is not going to work? Won't I have to update all the distributions that have 'vectors'?

Reply:
I'm not sure what other distributions have to do with this PR.  If their doc is messed up, that's a separate PR.  If there's a global explanation of this `vectors` notation, point to that.  If not, it needs to be explained so that users can figure out what the return type is for a function.

Reply:
There is global documentation for this convention at https://mc-stan.org/docs/functions-reference/vectorization.html